README.md

```diff
@@ -165,12 +165,12 @@ We outline our process below to clarify the roles of everyone involved.
 
 All pull requests must be approved by two committers before being merged into
 the repository. If any changes are necessary, the team will leave appropriate
-comments requesting changes to the code. Unfortunately we cannot guarantee a
+comments requesting changes to the code. Unfortunately, we cannot guarantee a
 pull request will be merged, even when modifications are requested, as the Elixir
 team will re-evaluate the contribution as it changes.
 
 Committers may also push style changes directly to your branch. If you would
-rather manage all changes yourself, you can disable the "Allow edits from maintainers"
+rather to manage all changes yourself, you can disable the "Allow edits from maintainers"
```

Comment:
As far as I can tell, no "to" here: https://www.gymglish.com/en/gymglish/english-grammar/would-rather

```suggestion
rather manage all changes yourself, you can disable the "Allow edits from maintainers"
```